cmd/generate/config/rules/kubernetes.go

```diff
@@ -0,0 +1,26 @@
+package rules
+
+import (
+	"github.com/zricethezav/gitleaks/v8/config"
+)
+
+func KubernetesSecret() *config.Rule {
+	// define rule
+	r := config.Rule{
+		RuleID:      "kubernetes-secret",
+		Description: "Possible Kubernetes Secret detected, posing a risk of leaking credentials/tokens from your deployments",
+		// We only match based on the "kind:secret" value of all kubernetes secrets
+		Regex: generateUniqueTokenRegex(`kind:\s*Secret`, true),
+
+		Keywords: []string{
+			"kind: Secret",
```

Comment:
This should just be "kind" or "secret", otherwise it won't match things like `kind:  Secret` (two spaces).

Reply:
Hey @rgmz, 

one of your examples e.g "flux.md" would have been catched by the path validation. But the other one would be a problem! 

I'm with you one the generic side. Is it possible to implement parsing logic with the libary e.g for yaml?

Reply:
@rgmz i tried to fix the corresponding keywords, but the tests are failing now. I ran `go generate`, could you please take a look?

Reply:
Can you share the specific error? I don't get any errors when running it.

```sh
$ go generate ./...
$ git diff
+++ b/config/gitleaks.toml
@@ -2220,8 +2220,9 @@ keywords = [
 id = "kubernetes-secret"
 description = "Possible Kubernetes Secret detected, posing a risk of leaking credentials/tokens from your deployments"
 regex = '''(?i)\b(kind:\s*Secret)(?:['|\"|\n|\r|\s|\x60|;]|$)'''
+path = '''(?i)\.ya?ml$'''
 keywords = [
-    "kind: secret",
+    "secret",
 ]
```

Reply:
Resolved it, just a typo on my side ;-)